charts/utils/templates/portforward.yaml

```diff
@@ -0,0 +1,54 @@
+{{ if .Values.portforwards.entities }}
+{{- range $portforward := .Values.portforwards.entities }}
+{{ $prefix := printf "%s-%s" $.Values.client $.Values.environment }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ $prefix }}-{{ $portforward.name }}
```

Comment:
I think you can define blocks like `{{ $prefix }}-{{ $portforward.name }}` as a variable and use `this variable` everywhere to not repeat it few times (the chance of error would be less then)

Reply:
Actually, `$prefix` is never used without joining with `$portforward.name`. So it makes sence to just adjust $prefix variable.